portfolio/src/main/webapp/scripts/api/index.js

```diff
@@ -1,9 +1,44 @@
 import { api } from './client.js';
+import { getCommentTemplate } from './utils.js';
 
-const addQuote = async () => {
-  const quote = await api.get('/data');
+const updateComments = async () => {
```

Comment:
Let's add comments on the methods here and other files: https://google.github.io/styleguide/jsguide.html#jsdoc

Reply:
Great! Let me work on that